ucm2/Intel/sof-essx8336/HiFi.conf

```diff
@@ -1,4 +1,37 @@
 SectionVerb {
+	EnableSequence [
+		# Disable all inputs / outputs
+		cset "name='Speaker Switch' off"
+		cset "name='Headphone Switch' off"
+		cset "name='Headset Mic Switch' off"
+		cset "name='Internal Mic Switch' off"
+		cset "name='DAC Mono Mix Switch' off"
+	]
+}
+
+BootSequence [
```

Comment:
The boot sequence belongs to the upper file (sof-essx8336.conf).

Reply:
Ok. moved it to sof-essx8336.conf.